en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md

```diff
@@ -9,7 +9,8 @@
  
 This [WSO2 EI 6.x product family](http://docs.wso2.com/enterprise-integrator) offers the conventional, centralized integration solution of the WSO2 middleware stack. WSO2 EI 6.x includes the ESB profile and other supporting profiles packaged in a single distribution.
  
-The new [WSO2 Enterprise Integrator (WSO2 EI 7.0.0)](https://ei.docs.wso2.com/en/latest/) is a hybrid platform that enables API-centric integration supporting various integration architecture styles: microservices architecture, cloud-native architecture, or a centralized ESB architecture. This integration platform offers a graphical/configuration-driven approach to developing integrations for any of the architectural styles.
+The new [WSO2 Enterprise Integrator (WSO2 EI 7.x)](https://ei.docs.wso2.com/en/latest/) is a hybrid platform that 
```

Comment:
Shall we point to a product page here or remove this link? Pointing the user to the doc landing page when they are in the docs already seems pointless.

Reply:
+1 to remove.